springql-core/src/connection.rs

```diff
@@ -26,13 +26,13 @@ fn setup_logger() {
 /// 1 stream pipeline has only 1 connection.
 /// In other words, the lifecycle of Connection and internal stream pipeline are the same.
 #[derive(Debug)]
-pub struct Connection {
+pub(crate) struct Connection {
```

Comment:
may be dont need `(crate)`

Reply:
@kazuk Correct. But If I change it into `pub`, I need to make many other structs `pub` due to `can't leak crate-private type` errors.

May I ask you to fix the visibility in https://github.com/SpringQL/SpringQL/pull/178 ?

Reply:
Oh, yes. field types are defined as `pub(crate)` . Connection can not be `pub`
I will fix in #178